packages/next/next-server/server/api-utils.ts

```diff
@@ -209,6 +210,26 @@ export function sendStatusCode(
   return res
 }
 
+/**
+ *
+ * @param res response object
+ * @param [statusOrUrl] `HTTP` status code of redirect
+ * @param url URL of redirect
+ */
+export function redirect(
+  res: NextApiResponse,
+  statusOrUrl: string | number,
+  url?: string
+): NextApiResponse<any> {
+  if (typeof statusOrUrl === 'string') {
+    url = statusOrUrl
+    statusOrUrl = 302
```

Comment:
The default should be 307, not 302.

Reply:
https://github.com/vercel/next.js/pull/14705#discussion_r447903171 @timneutkens @Timer please let me know what you want me to do.

Reply:
307 is better 👍 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307